Makefile.in

```diff
@@ -31,7 +31,7 @@ DEFS = -DHAVE_STDARG_H=@HAVE_STDARG_H@ \
 	-DHAVE_SYS_CYGWIN_H=@HAVE_SYS_CYGWIN_H@ \
 	-DHAVE_SIGNAL_H=@HAVE_SIGNAL_H@ \
 	$(DEFINES)
-INCS = -I. -I$(srcdir) $(INCLUDES)
```

Comment:
We may be able to do without the `-I$(srcdir)` but we need the `-I.` to be able to find the local headers.